### DISCUSSION TITLE
Unable to install OpenSSL on Windows 11

### DISCUSSION THREAD

**Discussion by fmeunierFKY** [openssl/openssl#24694] [Q&A]

**Comment by fmeunierFKY** [comment]
Hello, I'm unable to install OpenSSL on Windows 11....

here is the command
perl Configure  --prefix=c:\OpenSSL   no-apps no-argon2 no-aria no-asm no-async no-atexit no-autoalginit no-autoerrinit no-autoload-config no-bf no-blake2 no-bulk no-cached-fetch no-camellia no-capieng no-winstore no-cast no-chacha no-cmac no-cmp no-cms no-comp no-ct no-deprecated no-des no-dgram no-dh no-docs no-dsa no-dso no-dtls no-dynamic-engine no-ec no-ec2m no-ecdh no-ecdsa no-ecx no-engine no-err no-filenames no-gost no-http no-idea no-legacy no-loadereng no-makedepend no-md4 no-mdc2 no-module no-multiblock no-nextprotoneg no-ocb no-ocsp no-padlockeng no-pic no-pinshared no-poly1305 no-posix-io no-psk no-quic no-unstable-qlog no-rc2 no-rc4 no-rdrand no-rfc3779 no-rmd160 no-scrypt no-secure-memory no-seed no-shared no-siphash no-siv no-sm2 no-sm2-precomp no-sm3 no-sm4 no-sock no-srp no-srtp no-sse2 no-ssl no-ssl-trace no-static-engine no-stdio no-tests no-thread-pool no-threads no-tls no-integrity-only-ciphers no-ts no-ui-console no-uplink no-whirlpool no-tls1 no-tls1-method no-tls1_1 no-tls1_1-method no-tls1_2 no-tls1_2-method no-tls1_3 no-dtls1 no-dtls1-method no-dtls1_2 no-dtls1_2-method

**Comment by fmeunierFKY** [comment]
and here is the out-put

	"C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.40.33807\bin\HostX64\x64\nmake.exe" /                   depend
	"C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.40.33807\bin\HostX64\x64\nmake.exe" /                   _build_sw
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "-Mbuilddata" "util\dofile.pl" "-omakefile" "exporters\cmake\OpenSSLConfig.cmake.in" > OpenSSLConfig.cmake
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "-Mconfigdata" "-Mbuilddata" "util\dofile.pl" "-omakefile" "exporters\cmake\OpenSSLConfigVersion.cmake.in" > OpenSSLConfigVersion.cmake
	"c:\Perl\bin\perl.exe" "-I." "-Iutil\perl" "-Mconfigdata" "-MOpenSSL::paramnames" "util\dofile.pl" "-omakefile" "crypto\params_idx.c.in" > crypto\params_idx.c
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "-Minstalldata" "util\dofile.pl" "-omakefile" "exporters\cmake\OpenSSLConfig.cmake.in" > exporters\OpenSSLConfig.cmake
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "-Mconfigdata" "-Minstalldata" "util\dofile.pl" "-omakefile" "exporters\cmake\OpenSSLConfigVersion.cmake.in" > exporters\OpenSSLConfigVersion.cmake
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "-Minstalldata" "util\dofile.pl" "-omakefile" "exporters\pkg-config\libcrypto.pc.in" > exporters\libcrypto.pc
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "-Minstalldata" "util\dofile.pl" "-omakefile" "exporters\pkg-config\libssl.pc.in" > exporters\libssl.pc
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "-Mconfigdata" "-Mconfigdata" "-Minstalldata" "util\dofile.pl" "-omakefile" "exporters\pkg-config\openssl.pc.in" > exporters\openssl.pc
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\crypto\bn_conf.h.in" > include\crypto\bn_conf.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\crypto\dso_conf.h.in" > include\crypto\dso_conf.h
	"c:\Perl\bin\perl.exe" "-I." "-Iutil\perl" "-Mconfigdata" "-MOpenSSL::paramnames" "util\dofile.pl" "-omakefile" "include\internal\param_names.h.in" > include\internal\param_names.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\asn1.h.in" > include\openssl\asn1.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\asn1t.h.in" > include\openssl\asn1t.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\bio.h.in" > include\openssl\bio.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\cmp.h.in" > include\openssl\cmp.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\cms.h.in" > include\openssl\cms.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\comp.h.in" > include\openssl\comp.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\conf.h.in" > include\openssl\conf.h
	"c:\Perl\bin\perl.exe" "-I." "-Iutil\perl" "-Mconfigdata" "-MOpenSSL::paramnames" "util\dofile.pl" "-omakefile" "include\openssl\core_names.h.in" > include\openssl\core_names.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\crmf.h.in" > include\openssl\crmf.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\crypto.h.in" > include\openssl\crypto.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\ct.h.in" > include\openssl\ct.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\err.h.in" > include\openssl\err.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\ess.h.in" > include\openssl\ess.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\fipskey.h.in" > include\openssl\fipskey.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\lhash.h.in" > include\openssl\lhash.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\ocsp.h.in" > include\openssl\ocsp.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\opensslv.h.in" > include\openssl\opensslv.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\pkcs12.h.in" > include\openssl\pkcs12.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\pkcs7.h.in" > include\openssl\pkcs7.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\safestack.h.in" > include\openssl\safestack.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\srp.h.in" > include\openssl\srp.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\ssl.h.in" > include\openssl\ssl.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\ui.h.in" > include\openssl\ui.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\x509.h.in" > include\openssl\x509.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\x509_acert.h.in" > include\openssl\x509_acert.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\x509_vfy.h.in" > include\openssl\x509_vfy.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "util\dofile.pl" "-omakefile" "include\openssl\x509v3.h.in" > include\openssl\x509v3.h
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "-Mbuilddata" "util\dofile.pl" "-omakefile" "exporters\pkg-config\libcrypto.pc.in" > libcrypto.pc
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "-Mbuilddata" "util\dofile.pl" "-omakefile" "exporters\pkg-config\libssl.pc.in" > libssl.pc
	"c:\Perl\bin\perl.exe" "-I." "-Mconfigdata" "-Mbuilddata" "-Mbuilddata" "-Mbuilddata" "util\dofile.pl" "-omakefile" "exporters\pkg-config\openssl.pc.in" > openssl.pc
	"C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.40.33807\bin\HostX64\x64\nmake.exe" /                   depend
	"C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.40.33807\bin\HostX64\x64\nmake.exe" /                   _build_sw
	"cl"  /Zi /Fdossl_static.pdb /MT /Zl /Gs0 /GF /Gy /W3 /wd4090 /nologo /O2 -I"." -I"include" -I"providers\common\include" -I"providers\implementations\include" -D"L_ENDIAN" -D"OPENSSLDIR=\"C:\\Program Files\\Common Files\\SSL\"" -D"ENGINESDIR=\"C:\\OpenSSL\\lib\\engines-3\"" -D"MODULESDIR=\"C:\\OpenSSL\\lib\\ossl-modules\"" -D"OPENSSL_BUILDING_OPENSSL" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"NDEBUG"   -c /Focrypto\libcrypto-lib-params_idx.obj "crypto\params_idx.c"
params_idx.c
	"c:\Perl\bin\perl.exe" "-I." "-Iproviders\common\der" "-Mconfigdata" "-Mconfigdata" "-Mconfigdata" "-Moids_to_c" "util\dofile.pl" "-omakefile" "providers\common\include\prov\der_digests.h.in" > providers\common\include\prov\der_digests.h
	"c:\Perl\bin\perl.exe" "-I." "-Iproviders\common\der" "-Mconfigdata" "-Mconfigdata" "-Mconfigdata" "-Moids_to_c" "util\dofile.pl" "-omakefile" "providers\common\include\prov\der_rsa.h.in" > providers\common\include\prov\der_rsa.h
	"cl"  /Zi /Fdossl_static.pdb /MT /Zl /Gs0 /GF /Gy /W3 /wd4090 /nologo /O2 -I"providers\common\include\prov" -I"." -I"crypto" -I"include" -I"providers\implementations\include" -I"providers\common\include" -D"L_ENDIAN" -D"OPENSSLDIR=\"C:\\Program Files\\Common Files\\SSL\"" -D"ENGINESDIR=\"C:\\OpenSSL\\lib\\engines-3\"" -D"MODULESDIR=\"C:\\OpenSSL\\lib\\ossl-modules\"" -D"OPENSSL_BUILDING_OPENSSL" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"NDEBUG"   -c /Foproviders\common\der\libdefault-lib-der_rsa_sig.obj "providers\common\der\der_rsa_sig.c"
der_rsa_sig.c
	"cl"  /Zi /Fdossl_static.pdb /MT /Zl /Gs0 /GF /Gy /W3 /wd4090 /nologo /O2 -I"providers\common\include\prov" -I"." -I"crypto" -I"include" -I"providers\implementations\include" -I"providers\common\include" -D"L_ENDIAN" -D"OPENSSLDIR=\"C:\\Program Files\\Common Files\\SSL\"" -D"ENGINESDIR=\"C:\\OpenSSL\\lib\\engines-3\"" -D"MODULESDIR=\"C:\\OpenSSL\\lib\\ossl-modules\"" -D"OPENSSL_BUILDING_OPENSSL" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"NDEBUG"   -c /Foproviders\implementations\encode_decode\libdefault-lib-encode_key2any.obj "providers\implementations\encode_decode\encode_key2any.c"
encode_key2any.c
providers\implementations\encode_decode\encode_key2any.c(120): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
	"c:\Perl\bin\perl.exe" "-I." "-Iproviders\common\der" "-Mconfigdata" "-Moids_to_c" "-Moids_to_c" "util\dofile.pl" "-omakefile" "providers\common\include\prov\der_wrap.h.in" > providers\common\include\prov\der_wrap.h
	"cl"  /Zi /Fdossl_static.pdb /MT /Zl /Gs0 /GF /Gy /W3 /wd4090 /nologo /O2 -I"providers\common\include\prov" -I"." -I"crypto" -I"include" -I"providers\implementations\include" -I"providers\common\include" -D"L_ENDIAN" -D"OPENSSLDIR=\"C:\\Program Files\\Common Files\\SSL\"" -D"ENGINESDIR=\"C:\\OpenSSL\\lib\\engines-3\"" -D"MODULESDIR=\"C:\\OpenSSL\\lib\\ossl-modules\"" -D"OPENSSL_BUILDING_OPENSSL" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"NDEBUG"   -c /Foproviders\implementations\kdfs\libdefault-lib-x942kdf.obj "providers\implementations\kdfs\x942kdf.c"
x942kdf.c
providers\implementations\kdfs\x942kdf.c(218): warning C4267: '=': conversion from 'size_t' to 'uint32_t', possible loss of data
	"cl"  /Zi /Fdossl_static.pdb /MT /Zl /Gs0 /GF /Gy /W3 /wd4090 /nologo /O2 -I"providers\common\include\prov" -I"." -I"crypto" -I"include" -I"providers\implementations\include" -I"providers\common\include" -D"L_ENDIAN" -D"OPENSSLDIR=\"C:\\Program Files\\Common Files\\SSL\"" -D"ENGINESDIR=\"C:\\OpenSSL\\lib\\engines-3\"" -D"MODULESDIR=\"C:\\OpenSSL\\lib\\ossl-modules\"" -D"OPENSSL_BUILDING_OPENSSL" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"NDEBUG"   -c /Foproviders\implementations\signature\libdefault-lib-rsa_sig.obj "providers\implementations\signature\rsa_sig.c"
rsa_sig.c
providers\implementations\signature\rsa_sig.c(568): warning C4267: 'function': conversion from 'size_t' to 'unsigned int', possible loss of data
providers\implementations\signature\rsa_sig.c(593): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
providers\implementations\signature\rsa_sig.c(602): warning C4267: 'function': conversion from 'size_t' to 'unsigned int', possible loss of data
providers\implementations\signature\rsa_sig.c(661): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
providers\implementations\signature\rsa_sig.c(709): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
providers\implementations\signature\rsa_sig.c(749): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
providers\implementations\signature\rsa_sig.c(759): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
providers\implementations\signature\rsa_sig.c(789): warning C4267: 'function': conversion from 'size_t' to 'unsigned int', possible loss of data
providers\implementations\signature\rsa_sig.c(789): warning C4267: 'function': conversion from 'size_t' to 'unsigned int', possible loss of data
providers\implementations\signature\rsa_sig.c(821): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
providers\implementations\signature\rsa_sig.c(847): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
	"c:\Perl\bin\perl.exe" "-I." "-Iproviders\common\der" "-Mconfigdata" "-Mconfigdata" "-Mconfigdata" "-Moids_to_c" "util\dofile.pl" "-omakefile" "providers\common\der\der_digests_gen.c.in" > providers\common\der\der_digests_gen.c
	"cl"  /Zi /Fdossl_static.pdb /MT /Zl /Gs0 /GF /Gy /W3 /wd4090 /nologo /O2 -I"providers\common\include\prov" -I"crypto" -I"include" -I"providers\implementations\include" -I"providers\common\include" -D"L_ENDIAN" -D"OPENSSLDIR=\"C:\\Program Files\\Common Files\\SSL\"" -D"ENGINESDIR=\"C:\\OpenSSL\\lib\\engines-3\"" -D"MODULESDIR=\"C:\\OpenSSL\\lib\\ossl-modules\"" -D"OPENSSL_BUILDING_OPENSSL" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"NDEBUG"   -c /Foproviders\common\der\libcommon-lib-der_digests_gen.obj "providers\common\der\der_digests_gen.c"
der_digests_gen.c
	"c:\Perl\bin\perl.exe" "-I." "-Iproviders\common\der" "-Mconfigdata" "-Mconfigdata" "-Mconfigdata" "-Moids_to_c" "util\dofile.pl" "-omakefile" "providers\common\der\der_rsa_gen.c.in" > providers\common\der\der_rsa_gen.c
	"cl"  /Zi /Fdossl_static.pdb /MT /Zl /Gs0 /GF /Gy /W3 /wd4090 /nologo /O2 -I"providers\common\include\prov" -I"crypto" -I"include" -I"providers\implementations\include" -I"providers\common\include" -D"L_ENDIAN" -D"OPENSSLDIR=\"C:\\Program Files\\Common Files\\SSL\"" -D"ENGINESDIR=\"C:\\OpenSSL\\lib\\engines-3\"" -D"MODULESDIR=\"C:\\OpenSSL\\lib\\ossl-modules\"" -D"OPENSSL_BUILDING_OPENSSL" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"NDEBUG"   -c /Foproviders\common\der\libcommon-lib-der_rsa_gen.obj "providers\common\der\der_rsa_gen.c"
der_rsa_gen.c
	"cl"  /Zi /Fdossl_static.pdb /MT /Zl /Gs0 /GF /Gy /W3 /wd4090 /nologo /O2 -I"providers\common\include\prov" -I"crypto" -I"include" -I"providers\implementations\include" -I"providers\common\include" -D"L_ENDIAN" -D"OPENSSLDIR=\"C:\\Program Files\\Common Files\\SSL\"" -D"ENGINESDIR=\"C:\\OpenSSL\\lib\\engines-3\"" -D"MODULESDIR=\"C:\\OpenSSL\\lib\\ossl-modules\"" -D"OPENSSL_BUILDING_OPENSSL" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"NDEBUG"   -c /Foproviders\common\der\libcommon-lib-der_rsa_key.obj "providers\common\der\der_rsa_key.c"
der_rsa_key.c
	"c:\Perl\bin\perl.exe" "-I." "-Iproviders\common\der" "-Mconfigdata" "-Moids_to_c" "-Moids_to_c" "util\dofile.pl" "-omakefile" "providers\common\der\der_wrap_gen.c.in" > providers\common\der\der_wrap_gen.c
	"cl"  /Zi /Fdossl_static.pdb /MT /Zl /Gs0 /GF /Gy /W3 /wd4090 /nologo /O2 -I"providers\common\include\prov" -I"crypto" -I"include" -I"providers\implementations\include" -I"providers\common\include" -D"L_ENDIAN" -D"OPENSSLDIR=\"C:\\Program Files\\Common Files\\SSL\"" -D"ENGINESDIR=\"C:\\OpenSSL\\lib\\engines-3\"" -D"MODULESDIR=\"C:\\OpenSSL\\lib\\ossl-modules\"" -D"OPENSSL_BUILDING_OPENSSL" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"NDEBUG"   -c /Foproviders\common\der\libcommon-lib-der_wrap_gen.obj "providers\common\der\der_wrap_gen.c"
der_wrap_gen.c
	"lib" /nologo /out:libcrypto.lib @C:\Users\fmeun\AppData\Local\Temp\nmB3D1.tmp
	"lib" /nologo /out:providers\libcommon.lib @C:\Users\fmeun\AppData\Local\Temp\nmB568.tmp
	"lib" /nologo /out:providers\libdefault.lib @C:\Users\fmeun\AppData\Local\Temp\nmB5F6.tmp
	"c:\Perl\bin\perl.exe" "-I." -Mconfigdata "util\dofile.pl"  "-omakefile" "util\wrap.pl.in" > util\wrap.pl

**Comment by fmeunierFKY** [comment]
and nothing is produced to initial path c:\OpenSSL 

any help ...

thanks  a lot

franck

**Comment by nhorman** [comment]
did you run nmake install?

**Comment by fmeunierFKY** [comment]
just  nmake

**Reply by andrewkatson** [reply]
Do you have the part where there is an error? I've been running into problems building on Windows and its because I needed includes for C standard library defined by env variable `INCLUDE`. And then C standard binaries on `PATH`. And C standard libraries in an env variable called `LIB`. I wonder if any of these are possibly the culprit here.

**Reply by nhorman** [reply]
I'm not sure what you mean by that.  Openssl doesn't require setting the environment variables INCLUDE or LIB independently at all.

And looking back at the output you posted, it shows no error.  So its really difficult to tell whats going on here.

Looking at your configure line, it also appears you may have disabled everything in the library.

if you run:
```
perl Configure
nmake
nmake DESTDIR=c:\OpenSSL install
```

is there any output produced in c:\OpenSSL?

**Reply by andrewkatson** [reply]
I think it might need some now on Windows because other people have had trouble with `nmake` finding where the C standard library is. https://github.com/openssl/openssl/discussions/24815. My guess is that when they installed their Visual Studio Build or Windows Kit it didn't set the environment variables for them like it normally does.
